.github/workflows/release.yml

```diff
@@ -0,0 +1,51 @@
+name: Release Targets
+
+on:
+  create:
+    tags:
+      - .*
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./frontend
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+        cache: 'npm'
+        cache-dependency-path: frontend/package-lock.json
+    - run: npm install
+    - run: npm run build --if-present
+    - name: Zip manifest.json
```

Comment:
Here I would additionally create an artifact with the compiled web app, because it would be useful for the deployment. It is also part of the acceptance criteria for the ticket.

Reply:
Alright I fixed that and made a new tag release with it check it out here 
https://github.com/amosproj/amos2021ws03-teams-to-nextcloud/releases/tag/untagged-4559e4c3086623088b58